backend/api/Areas/Organizations/Controllers/OrganizationController.cs

```diff
@@ -30,19 +32,36 @@ public class OrganizationController : ControllerBase
         /// <summary>
         /// Creates a new instance of a PersonController class, initializes it with the specified arguments.
         /// </summary>
+        /// <param name="pimsService"></param>
         /// <param name="pimsRepository"></param>
         /// <param name="mapper"></param>
         ///
-        public OrganizationController(IPimsRepository pimsRepository, IMapper mapper)
+        public OrganizationController(IPimsService pimsService, IPimsRepository pimsRepository, IMapper mapper)
         {
+            _pimsService = pimsService;
             _pimsRepository = pimsRepository;
             _mapper = mapper;
         }
         #endregion
 
         #region Endpoints
         /// <summary>
-        /// Get the contact for the specified primary key 'id'.
+        /// Get the organization for the specified primary key 'id'.
+        /// </summary>
+        /// <returns></returns>
+        [HttpGet("{id:long}")]
+        [HasPermission(Permissions.ContactView)]
+        [Produces("application/json")]
+        [ProducesResponseType(typeof(Models.Organization.OrganizationModel), 200)]
+        [SwaggerOperation(Tags = new[] { "organization" })]
+        public IActionResult GetOrganization(int id)
+        {
+            var organization = _pimsService.OrganizationService.GetOrganization(id);
+            return new JsonResult(_mapper.Map<Models.Organization.OrganizationModel>(organization));
+        }
+
+        /// <summary>
+        /// Add the organization to the datastore.
```

Comment:
Did you rebased your branch after my PR got merged to dev? Wouldn't expect to see this change here... Not related to this PR